### DISCUSSION TITLE
如果能去掉主密码就好了

### DISCUSSION THREAD

**Discussion by mlsqq** [kingToolbox/WindTerm#482] [Ideas]
我提议可以本地保存密码以免每次登录输入，去掉主密码（ps：懒人）
配置免密登录后直接点击登录，无需输入账户
然后就是修改文件，本地ssh-keygen偶尔窗口无响应可以点，但是没作用
总体来说好用，就是密码这一块确实每次输入有点麻烦，加油！！！！作者

**Comment by Lemonawa** [comment]
Translate by DeepL:
# It would be nice if the master password could be removed.
I propose to save the password locally to avoid entering it every time you log in and remove the master password (ps: lazy people)
After configuring the password-free login, you can directly click on the login, no need to enter the account
Then is to modify the file, local ssh-keygen occasionally window does not respond to point, but no effect
Overall good, is the password piece is really a little trouble to enter each time, cheer ！！！！ Author



Please use English,OK?

**Reply by jeffmasc** [reply]
I wish this features either, Just a screen password enough( and can skip either, bocs I have a windows lock screen pwd).

**Comment by kingToolbox** [comment]
@Lemonawa
Thank you.

@yangyang0223 
Sorry for the late reply, I must have missed this.

It is possible to bypass the master password. And you need to set the master password to an empty string to turn it off. Since you have already set a master password, please follow [Change Master Password ](https://kingtoolbox.github.io/2021/04/23/change-master-password/) to reset the password to an empty string.

**Reply by zhoujun0601** [reply]
it works, but it looks a liitle confused. Hope you can add a deletion selection.

**Reply by speakchan** [reply]
you can modify the \.wind\profiles\default.v10\user.config ,"application.masterPassword" : false to avoid master password
